documentation/secure_storage.md

```diff
@@ -230,9 +230,10 @@ or crypto unit according to the method defined by your SoC vendor.
 An attacker can backup the whole `/data/tee` folder and restore it at later
 time.
 
-The basic idea of detecting rollback attack is to store a hash representing the
-state of `/data/tee/dirf.db` into another storage which has anti-rollback
-capability such as the eMMC RPMB partition.
+The basic idea of detecting rollback attack is to store information
+representing the state of `/data/tee/dirf.db` into another storage which has
+anti-rollback capability such as the eMMC RPMB partition. The hash tree counter
```

Comment:
OK, it's not the purpose of this doc to discuss the design of a feature that we don't support yet. I'll drop the last sentence.